README.md

```diff
@@ -6,36 +6,67 @@ A tool for validating community plugins for publishing to Grafana.com.
 
 The tool expects path to either a remote or a local ZIP archive.
 
-## Install
+## Install and usage
+
+### With docker (recommended\*)
 
 ```SHELL
-cd pkg/cmd/plugincheck2
-go install
+docker run --pull=always grafana/plugin-validator-cli -sourceCodeUri [source_code_location/] [plugin_archive.zip]
+```
+
+Example:
+
+```SHELL
+docker run --pull=always grafana/plugin-validator-cli -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
 ```
 
-## Run
+> \* Docker is recommended because the image contains all the security scanning tools for the validator such as semgrep, gosec and osv-scanner
 
-Typically running the checker with default settings is the easiest method to see if there are issues with a plugin.
+### With NPX
 
 ```SHELL
-plugincheck2 -config config/default.yaml https://github.com/marcusolsson/grafana-jsonapi-datasource/releases/download/v0.6.0/marcusolsson-json-datasource-0.6.0.zip
+npx -y @grafana/plugin-validator -sourceCodeUri [source_code_location/] [plugin_archive.zip]
 ```
 
-To wrap the output with another tool (like the validator ui), running with the `terse-json.yaml` config can be used.
+Example:
+
+```SHELL
+nnpx -y @grafana/plugin-validator -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zippx -y @grafana/plugin-validator -sourceCodeUri
```

Comment:
typo: nnpx

Reply:
also _think_ there might be some extra stuff at the end, and you just wanted: `npx -y @grafana/plugin-validator -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip` ?

Reply:
Thanks! I missed this.